src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotCorrectedObject = {};
+  const swappedValues = {};
+
+  for (const key in robot) {
+    if (robot.hasOwnProperty(key)) {
```

Comment:
The 'hasOwnProperty' check is not necessary, as using 'for...in' already checks for properties in the object.